core/src/main/scala/caliban/wrappers/Wrapper.scala

```diff
@@ -67,7 +67,7 @@ object Wrapper {
    * If false, simple pure values will be ignored.
    */
   case class FieldWrapper[R](
-    f: (ZQuery[R, Nothing, ResponseValue], FieldInfo) => ZQuery[R, CalibanError, ResponseValue],
+    f: (ZQuery[R, CalibanError, ResponseValue], FieldInfo) => ZQuery[R, CalibanError, ResponseValue],
```

Comment:
We could restrict to `ExecutionError` I think?

Reply:
I tried but wrapper returns a `CalibanError`, should I make that return `ExecutionError` too?

Reply:
Ah no you're right.